app/database/migrations/20210213121940_create_clients_table.js

```diff
@@ -0,0 +1,20 @@
+
+const status = [
+    'SEND_EMAILS',
+    'NO_EMAILS'
+];
+
+exports.up = function(knex) {
+    return knex.schema.createTable('clients', table => {
+        table.integer('job').unsigned().notNullable();
+        table.string('first_name');
+        table.string('last_name');
+        table.text('email');
+        table.enum('status', status, {useNative: true, enumName:'status_enum'}).notNullable().index();
```

Comment:
actually you know what, since there are only 2 condition, let's just make this a boolean called is_sending. or something like that

Reply:
done